cli/internal/nodeprovider/local/createNodeIfNotExists_test.go

```diff
@@ -2,15 +2,19 @@ package local
 
 import (
 	"context"
+	"io/ioutil"
+
 	. "github.com/onsi/ginkgo"
-	"os"
+	. "github.com/onsi/gomega"
 )
 
 var _ = Context("CreateNodeIfNotExists", func() {
 	It("shouldn't panic", func() {
 		/* arrange */
+		dataDir, err := ioutil.TempDir("", "")
+		Expect(err).To(BeNil())
```

Comment:
Not a big deal but you might consider panic'ing if these fail rather than failing the test. Thinking being we're not testing if ioutil.TempDir errs; if it does then we can't test so panic seems appropriate.

Reply:
Sure, that's fine. I wasn't quite sure what the preferred pattern should be.